studio-docs/source/org/members.mdx

```diff
@@ -5,53 +5,65 @@ sidebar_title: Members and roles
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `Organization Admin` | Has management access to the entire organization, including its members, graphs, and configuration.                                                                                                                |
+| `Graph Admin`        | Has management access to the organization's graphs, but not to members or organization-level configuration. Intended for developers who are considered admins to the graph.                                        |
+| `Contributor`        | Can push new schemas to graphs, but cannot configure graph settings or integrations. Intended for back-end developers who are authorized to make changes to graphs. _Cannot_ push schema changes to [protected variants](./graphs/#protected-variants-enterprise-only).                                               |
+| `Observer`           | Has view-only access to the organization's graph data, such as schemas and metrics. Can also execute queries in the Explorer. Intended for back-end developers who are _not_ authorized to make changes to graphs. |
+| `Consumer`           | Has view-only access to the organization's graph _schemas_, but not metrics. Can also execute queries in the Explorer. Intended for application developers building clients against a graph.                       |
+| `Billing Manager`    | Has management access to organization-level configuration and billing. Can also remove members (but not invite them). No access to graphs.                                                                         |
+
+Only members with the `Organization Admin` role can assign roles to other members. You can see which role you have in a particular organization from your [user settings page](https://studio.apollographql.com/user-settings), or from the organization's Members tab.
+
+## Graph-specific member roles
+
+You can override a member's organization-wide role for individual graphs. For example, if a member has the `Observer` role in your organization, you can assign them the `Contributor` role for a graph that they need extra access to.
+
+> A member's graph-specific role must have _more_ permissions than their organization-wide role.
+
+Members with the `Graph Admin` or `Organization Admin` role can assign graph-specific roles from the Access tab of the graph's Settings page.
+
+## Role permissions
+
+> We recently added the `Graph Admin` role and reduced permissions for members with the `Contributor` role.
+
+| Action                                                 | Org Admin | Graph Admin | Billing Manager | Contributor | Observer | Consumer |
```

Comment:
I'd consider moving Billing Manager to be either the first or last column, or frankly just having it have its own section listing the small number of things it can do.

Reply:
Oh yeah, great idea